zerver/lib/test_helpers.py

```diff
@@ -308,19 +338,21 @@ def assert_json_success(self, result):
         return json
 
     def get_json_error(self, result, status_code=400):
+        # type: (HttpResponse, int) -> Any
```

Comment:
I think we know the return value will be a Dict[str, Any], just from reading the code here.